Classes/YLRefreshHeaderView/YLRefreshHeaderView.m

```diff
@@ -61,7 +61,7 @@ - (CGFloat)pullAmountToRefresh {
 
 #pragma mark UIScrollViewDelegate
 
-- (void)containingScrollViewDidScroll:(UIScrollView *)scrollView {
+- (void)containingScrollViewDidScroll:(nonnull UIScrollView *)scrollView {
```

Comment:
Shall this be a rule then to replicate the nullability annotations in the `.m` files? It makes sense but I'm wondering as it's hard to enforce and I also saw that AFNetworking and other libraries didn't do it.
@bmelts, @ytok : Any thoughts on that?